### PR TITLE
Added menu item with shortcut for dev tools in production

### PR DESCRIPTION
**Pull request by kodaman2**
As per @lnrdnl [comment](https://github.com/lauthieb/code-notes/issues/119#issuecomment-563919412) I've went ahead and added a menu item to launch dev tools in the production app. This will help in the long run to debug app from a user standpoint. This used to be on by default, but was disabled at some point by electron package hence the menu item now.

CTRL/CMD+SHIFT+I, it also under Application/Dev Tools menu

### CODE REVIEW

File: src/main/index.js [hunk 1]
```diff
@@ -44,6 +44,13 @@ function createWindow() {
           selector: 'orderFrontStandardAboutPanel:',
         },
         { type: 'separator' },
+        {
+          label: 'Dev Tools',
```
Comment (lauthieb) [inline src/main/index.js]:
```suggestion
          label: 'Toggle dev tools',
```

File: src/main/index.js [hunk 2]
```diff
@@ -44,6 +44,13 @@ function createWindow() {
...
+          accelerator: 'CmdOrCtrl+Shift+I',
+          click: () => {
+            mainWindow.openDevTools();
```
Comment (lauthieb) [inline src/main/index.js]:
```suggestion
            mainWindow.toggleDevTools();
```

**Comment by kodaman2** [comment]
@lauthieb changes proposed commited.

**Comment by lauthieb** [comment]
Merged :tada:

**Pull request merged**